### PR TITLE
content: Add Platform support package, distinct from Basic

### PR DESCRIPTION
**Pull request by mogul**
## Changes proposed in this pull request:
- Add now-default "Platform" support package, which includes 10hrs of support.
- Explicitly list "Basic" as having been for previous contracts.

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/update-support-packages/)

## Security Considerations
None, this is only an update in text.

### CODE REVIEW

**Comment by bengerman13** [comment]
I assume the discussion on these names happened elsewhere and is finalized?

**Comment by bengerman13** [comment]
also, maybe out of scope for your PR, but this looks kinda bad to me:
image

**Review by bengerman13 — APPROVED** [review]
I don't like that we're changing the name of the default, and adding a new option reusing the old name. Assuming that's out of scope for this conversation, this works well enough

File: _data/support-packages.yml [hunk 1]
```diff
@@ -1,24 +1,36 @@
-- package_name: Basic Support
+- package_name: Platform Support
   description: >
-    This is the support level of our existing cloud.gov agreements.
-    If you run into any issues with the underlying platform, we will
-    resolve them at no cost.
+    This is the support level for previous cloud.gov agreements. If you run into
```
Comment (bengerman13) [inline _data/support-packages.yml:3]:
```suggestion
    (previously "Basic Support") If you run into
```

**Comment by mogul** [comment]
> I don't like that we're changing the name of the default, and adding a new option reusing the old name. 

Same for me. 

> Assuming that's out of scope for this conversation, this works well enough

@melanienleopold had me copying in names and descriptions from the `Estimated Cost Breakdown` section of [this doc](https://docs.google.com/document/d/1Rva2YY7xaSBQJfjHUdz8-E_SMv8uIoTQ/edit). I have no idea how much effort led up to those changes or how much flexibility remains.

**Comment by mogul** [comment]
Based on our meeting to discuss, more changes will be coming before we merge this. @melanienleopold can you summarize the changes we need to make?

**Comment by markdboyd** [comment]
Superseded by #2161
